src/items.c

```diff
@@ -25,6 +25,33 @@
 
 #include "g_local.h"
 
+enum
+{
+	ITEM_CLASS_HEALTH,
```

Comment:
we should initialize this to 0

Reply:
enumerations _always_ start at 0 if a constant is not defined... it's in the standard

Reply:
MISRA C:2004, 9.3 - _In an enumerator list, the "=" construct shall not be used to explicitly initialize members other than the first, unless all items are explicitly initialized._ 

It is an industry standard best practice...

Reply:
It's not industry standard, because you can introduce mistakes that way, giving multiple items the same value.

Here is from the C99 standard: _If the first enumerator hasno=,the value of its enumeration constant is 0.Each subsequent enumerator with no=defines  its  enumeration  constant  as  the  value  of  the  constant  expression  obtained  byadding 1 to the value of the previous enumeration constant.(The use of enumerators with=may produce enumeration constants with values that duplicate other values in the sameenumeration.)_

Reply:
best practice != standard.
Misra is on top of the standard, which is a collection of rules/suggestions that could guarantee that you write safer/portable code.
I know that what you say is true: the first enum is auto-initiated to 0.

But still, the Misra guideline suggest that you should explicitly initiate it to 0.

So you are not a Misra guy, huh? :)

Reply:
I am not a consortium guy, in general. Design by committee tends to be inferior.